src/ontology/uberon-edit.obo

```diff
@@ -222010,6 +222010,15 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:0003687 ! foramen magnum
 property_value: depicted_by http://locatelambda.org/wp-content/uploads/2018/02/opisthion.jpg xsd:string
 
+[Term]
+id: UBERON:7500120
+name: zygion
+def: "An anatomical point on the most lateral point of the zygomatic arch." [ISBN: 9004086161\, 9789004086166]
```

Comment:
Consider for brevity:

```suggestion
def: "The most lateral anatomical point of the zygomatic arch." [ISBN: 9004086161\, 9789004086166]
```

Unable to access the provided ISBN, but the definition aligns with <a href="https://medical-dictionary.thefreedictionary.com/zygion">zygion</a>